AnalogInput potentiometer
DigitalVector3 button 